test_files/coerce/coerce.js.transformer

```diff
@@ -0,0 +1,12 @@
+goog.module('test_files.coerce.coerce');var module = module || {id: 'test_files/coerce/coerce.js'};/**
```

Comment:
@evmar LMK what you think of this - while the `.patch` files lack context to review them, these here lack clarity in what changed. Not sure what I prefer, but it'll be easy to reinstate the diffing method for these.